content-src/asrouter/templates/Trailhead/_Trailhead.scss

```diff
@@ -401,9 +396,14 @@
   }
 }
 
-// If the window is too short, we need to allow scrolling so user can get to Start Browsing button.
-@media (max-height: 700px) {
+// If the window is too short or narrow, we need to allow scrolling so user can get to Start Browsing button.
+@media (max-height: 700px), (max-width: 765px) {
```

Comment:
Where did we get this 765px?

Reply:
I made my height just above 700px (so it doesn't trigger the first case) and reduced the width until the button goes off the bottom. I set it to a little over that so that hopefully German will fit too.

Reply:
Set `intl.l10n.pseudo` to "accented" to test longer strings !

Reply:
oh wow... both these values need increasing !